deploy/requirements-test.txt

```diff
@@ -9,4 +9,5 @@ tavern==1.19.0
 fakeredis==1.5.0
 pytest-env==0.6.2
 responses==0.13.0
-pytest-xdist~=2.5.0
\ No newline at end of file
+pytest-xdist~=2.5.0
+importlib-metadata==4.13.0; python_version == '3.7'
```

Comment:
Why do you add this library line here?

Reply:
I did not add this one. 

Should line #13 be removed?

Reply:
that was added by @triklozoid 
tests were failing on python 3.7

Reply:
@makseq ^^